test/parallel/test-crypto-hmac.js

```diff
@@ -11,9 +11,24 @@ var crypto = require('crypto');
 // Test HMAC
 var h1 = crypto.createHmac('sha1', 'Node')
                .update('some data')
-               .update('to hmac')
-               .digest('hex');
-assert.equal(h1, '19fd6e1ba73d9ed2224dd5094a71babe85d9a892', 'test HMAC');
+               .update('to hmac');
+assert.equal(h1.digest('hex'),
+             '19fd6e1ba73d9ed2224dd5094a71babe85d9a892',
+             'test HMAC');
+
+var h2 = crypto.createHmac('sha1', 'Node')
+               .update('some data')
+               .update('to hmac');
+assert.ok(h2.validate(
+  new Buffer('19fd6e1ba73d9ed2224dd5094a71babe85d9a892', 'hex'),
```

Comment:
These should now use `Buffer.from('19fd6e1ba73d9ed2224dd5094a71babe85d9a892', 'hex')`